components/dropdown/dropdown-content-styles.js

```diff
@@ -1,285 +1,288 @@
 import '../colors/colors.js';
 import { css } from 'lit-element/lit-element.js';
 
-export const dropdownContentStyles = css`
-
-	:host {
-		--d2l-dropdown-above-animation-name: d2l-dropdown-above-animation;
-		--d2l-dropdown-animation-name: d2l-dropdown-animation;
-		--d2l-dropdown-background-color: #ffffff;
-		--d2l-dropdown-border-color: var(--d2l-color-mica);
-		--d2l-dropdown-foreground-color: var(--d2l-color-ferrite);
-		--d2l-dropdown-shadow-color: rgba(0, 0, 0, 0.15);
-		box-sizing: border-box;
-		color: var(--d2l-dropdown-foreground-color);
-		display: none;
-		left: 0;
-		position: absolute;
-		text-align: left;
-		top: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
-		width: 100%;
-		z-index: 1000; /* position on top of floating buttons */
-	}
-
-	:host([theme="dark"]) {
-		--d2l-dropdown-above-animation-name: d2l-dropdown-above-animation-dark;
-		--d2l-dropdown-animation-name: d2l-dropdown-animation-dark;
-		--d2l-dropdown-background-color: #333536; /* ferrite @ 70% */
-		--d2l-dropdown-border-color: var(--d2l-color-ferrite);
-		--d2l-dropdown-foreground-color: var(--d2l-color-sylvite);
-		--d2l-dropdown-shadow-color: rgba(0, 0, 0, 1);
-		opacity: 0.9;
-	}
-
-	@media (min-width: 616px) {
-		:host([opened]) {
-			animation: var(--d2l-dropdown-animation-name) 300ms ease;
+export function dropdownContentStyles(breakpoint) {
+	// use default mobile breakpoint at 616px if no override provided
+	return css`
+
+		:host {
+			--d2l-dropdown-above-animation-name: d2l-dropdown-above-animation;
+			--d2l-dropdown-animation-name: d2l-dropdown-animation;
+			--d2l-dropdown-background-color: #ffffff;
+			--d2l-dropdown-border-color: var(--d2l-color-mica);
+			--d2l-dropdown-foreground-color: var(--d2l-color-ferrite);
+			--d2l-dropdown-shadow-color: rgba(0, 0, 0, 0.15);
+			box-sizing: border-box;
+			color: var(--d2l-dropdown-foreground-color);
+			display: none;
+			left: 0;
+			position: absolute;
+			text-align: left;
+			top: calc(100% + var(--d2l-dropdown-verticaloffset, 20px));
+			width: 100%;
+			z-index: 1000; /* position on top of floating buttons */
+		}
+
+		:host([theme="dark"]) {
+			--d2l-dropdown-above-animation-name: d2l-dropdown-above-animation-dark;
+			--d2l-dropdown-animation-name: d2l-dropdown-animation-dark;
+			--d2l-dropdown-background-color: #333536; /* ferrite @ 70% */
+			--d2l-dropdown-border-color: var(--d2l-color-ferrite);
+			--d2l-dropdown-foreground-color: var(--d2l-color-sylvite);
+			--d2l-dropdown-shadow-color: rgba(0, 0, 0, 1);
+			opacity: 0.9;
+		}
+
+		@media (min-width: ${breakpoint}px) {
```

Comment:
We sub in the provided breakpoint in these media queries, rather than a css variable. Note that I didn't change any content here, even though the diff looks pretty crazy...